package.json

```diff
@@ -25,7 +25,7 @@
     "dagre": "^0.8.5",
     "eslint-plugin-only-error": "^1.0.2",
     "fhir-mapper": "^3.1.1",
-    "fhir-visualizers": "^0.0.1",
+    "fhir-visualizers": "~0.0.2",
```

Comment:
Maybe my understanding of this syntax is incorrect but shouldn't `^` have installed 0.0.2 since it is a patch?

Reply:
I think you're correct @blangley28 from [looking at the docs](https://github.com/npm/node-semver#tilde-ranges-123-12-1).  Not sure one is better than the other, but I suppose `~` is a little more conservative.

Reply:
I seem to recall there's something special about the versioning for initial patch releases. 0.0.1 -> 0.0.2 is treated differently than 1.1.1 -> 1.1.2 for example. Not sure about the difference between `^` and `~` but either way I think the manual bump to 0.0.2 is necessary.

Reply:
I see, that's interesting... Did we want `^0.0.2` or `^~0.0.2`?

Reply:
Is `^~0.0.2` valid syntax? I think `^0.0.2` should do it